core/runtime/runtime.go

```diff
@@ -12,3 +12,9 @@ import (
 func WithCaptureHTTPResponse(parent context.Context, resp **http.Response) context.Context {
 	return context.WithValue(parent, config.ContextHTTPResponse, resp)
 }
+
+// WithCaptureHTTPRequest adds the raw HTTP request retrieval annotation to the parent context.
+// The req parameter will contain the raw HTTP request made to the API.
```

Comment:
```suggestion
// After the request has completed, the req parameter will contain the raw HTTP request made to the API.
```

Reply:
I would still highlight the `req` variable will only be populated once the request is executed